dotcom-rendering/src/web/lib/renderContainer.tsx

```diff
@@ -0,0 +1,53 @@
+import { ContainerLayout } from '../components/ContainerLayout';
+import { DynamicFast } from '../components/DynamicFast';
+import { FixedLargeSlowXIV } from '../components/FixedLargeSlowXIV';
+
+type Props = {
+	trails: DCRFrontCard[];
+	containerType: DCRContainerType;
+};
+
+const renderContainer = ({ trails, containerType }: Props) => {
+	switch (containerType) {
+		case 'dynamic/fast':
+			return <DynamicFast trails={trails} />;
+		case 'fixed/large/slow-XIV':
+			return <FixedLargeSlowXIV trails={trails} />;
+		default:
+			// TODO: This default allows us to render fronts in-development where we might
+			// not support all the container types, but it should be removed / re-investigated
+			// before fronts are released
+			return <DynamicFast trails={trails} />;
+	}
+};
+
+export const renderFrontContainer = ({
+	collection,
+	showTopBorder,
+}: {
+	collection: DCRCollectionType;
+	showTopBorder: boolean;
+}) => {
+	// TODO: We also need to support treats containers
+	// Backfills should be added to the end of any curated content
+	const trails = collection.curated.concat(collection.backfill);
+	// There are some containers that have zero trails. We don't want to render these
+	if (trails.length === 0) return null;
```

Comment:
Should we log this as some kind of error?

Reply:
I don't think this would be an error? If it's possible to set these in fronts tool we probably want to treat it as expected behaviour

Reply:
I might be a little unfamiliar with the lingo. What’s the purpose of a container with no trails?

Reply:
I think there is zero purpose. But the tool and, consequently, the model allow it so we need to have this line. But I don't think we should render anything if there are no trails.

Perhaps historically there have been examples of people using containers as vehicles for weird css but I don't think we should support this pattern.